okhttp-logging-interceptor/src/main/java/okhttp3/logging/utf8.kt

```diff
@@ -13,6 +13,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+@file:JvmName("UTF8")
```

Comment:
Nice catch, we should have some test that covers this public API.

Reply:
This name isn't hidden from Java.

Reply:
My preference is to do this as we do in Okio.
https://github.com/square/okio/blob/master/okio/src/jvmMain/kotlin/okio/-Platform.kt#L17

Reply:
- Got it. Looks like the file name could be `-UTF8.kt`?
- Seems like test is nicely written in `HttpLoggingInterceptor` and `IsProbablyUtf8Test` :)